src/plugins/fs/ext.c

```diff
@@ -59,6 +59,76 @@ static guint32 fs_mode_util[BD_FS_MODE_LAST+1] = {
 
 #define UNUSED __attribute__((unused))
 
+static gint8 compute_percents(guint8 pass_cur, guint8 pass_total, gint val_cur, gint val_total) {
```

Comment:
Please add a space between function names an parentheses. In definitions as well as where the functions are called.